src/main/java/com/netscape/management/client/util/Debug.java

```diff
@@ -423,7 +423,8 @@ public static void println(String type, String s) {
                 System.err.println(s);
             }
             if (_fPrintCallStack) {
-                Thread.currentThread().dumpStack();
+                Thread.currentThread();
```

Comment:
I suppose the `Thread.currentThread()` itself (and similar code below) is redundant.

Reply:
Indeed, I imagine there is plenty to scrape out if one were to look deep into the abyss!